src/components/elements/CardComponent.vue

```diff
@@ -23385,6 +23385,108 @@ export default {
     getTags() {
       return this.model.Tags.join(' - ').toUpperCase() || ''
     },
+    generateRulesText() {
+      let deeper = element => {
+        if (typeof element === "object")
+          return element[R.keys(element)[0]]
+        else 
+          return false
+      }
+      let decapital = x => {
+        return R.join("", R.over(R.lensIndex(0), R.toLower, x))
+      }
+      let effectsToText = Effects => {
+        let effectText = []
+        Effects.forEach(effect => {
+          let effectKeyword = R.keys(effect)[0]
+          let interaction = getInteractionText(effectKeyword)
+          
+          let interactionBlock = []
+          interaction.forEach(element => {
+            if (R.includes("§", element)) {
+              let pure = R.join("", R.without(["§", "."], element))
+
+              let interactionNode = effect[effectKeyword][pure]
+
+              if (getRule(pure)) {
+                if (getRule(pure).type == "interface") {
+                  let picked = R.keys(interactionNode)[0]
+                  interactionBlock.push(getRule(picked).interactionText)
+                }
+                else
+                  console.log("pure interaction?", getInteractionText(pure))
+              }
+              else {
+                let nextNode = deeper(interactionNode) 
+
+                if (nextNode) {
+                  interactionBlock.push(nextNode)
+                }
+                else {
+                  interactionBlock.push(interactionNode)
+                }    
+              }
+            } 
+            else
+              interactionBlock.push(element)
+          })
+          effectText.push(R.join(" ", interactionBlock))
+        })
+        return effectText
+      }
+
+      let rules = this.cardRules.definitions
+      let getRule = key => this.cardRules.definitions[decapital(key)]
+      let getInteractionText = key => R.split(" ", rules[decapital(key)].interactionText)
+
+      let rulesText = ""
```

Comment:
```suggestion
      let rulesText = []
```

Please empty array not empty string.
this method is supposed to return an array, not a string